source/whats-new.txt

```diff
@@ -22,6 +22,305 @@ What's New in 4.0
 
 New features of the 4.0 Node.js driver release include:
 
+TypeScript
+~~~~~~~~~~
+
+- We've migrated the driver to TypeScript. You can now harness the type
+  hinting and intellisense features in editors that support it to develop
+  your MongoDB applications. Enjoy the benefits of this work in pure JavaScript
+  projects as well!
```

Comment:
Nit: I think there is something about using exclamation points in the style guide, and the last statement doesn't seem to add any value.

Reply:
I agree the exclam can be removed, but the last statement adds a lot of value to any user using an editor with intellisense in a JavaScript project.